qcschema/dev/properties/cc_properties.py

```diff
@@ -0,0 +1,101 @@
+"""
+The complete list of Coupled-Cluster properties.
+"""
+
+cc_properties = {}
+
+cc_properties["ccsd_same_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The portion of CCSD doubles correlation energy from same-spin (i.e. triplet) correlations, without any user scaling."
+}
+
+cc_properties["ccsd_opposite_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The portion of CCSD doubles correlation energy from opposite-spin (i.e. singlet) correlations, without any user scaling."
+}
+
+cc_properties["ccsd_singles_energy"] = {
+    "type": "number",
+    "description": "The singles portion of the CCSD correlation energy. Zero except in ROHF."
+}
+
+cc_properties["ccsd_doubles_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The doubles portion of the CCSD correlation energy including same-spin and opposite-spin correlations."
+}
+
+cc_properties['ccsd_correlation_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The CCSD correlation energy."
+}
+
+cc_properties['ccsd_total_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The total CCSD energy (CCSD correlation energy + HF energy)."
+}
+
+cc_properties['ccsd_t_correlation_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The CCSD(T) correlation energy."
+}
+
+cc_properties['ccsd_t_total_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The total CCSD(T) energy (CCSD(T) correlation energy + HF energy)."
+}
+
+cc_properties['ccsdt_correlation_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The CCSDT correlation energy."
+}
+
+cc_properties['ccsdt_total_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The total CCSDT energy (CCSDT correlation energy + HF energy)."
+}
+
+cc_properties['ccsdtq_correlation_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The CCSDTQ correlation energy."
+}
+
+cc_properties['ccsdtq_total_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The total CCSDTQ energy (CCSDTQ correlation energy + HF energy)."
+}
+
+cc_properties["cc_dipole_moment"] = {
```

Comment:
my fault for using shorthand. I'd vote for separate `ccsd_dipole_moment`, `ccsdt_dipole_moment`, etc. Not a problem that these aren't usually generated in an energy calc -- they're just in reserve.

Reply:
Okay sounds good, I'll update it so that there is a separate dipole moment for each method.